spectrum/test_article.py

```diff
@@ -130,6 +130,19 @@ def test_article_with_unicode_content(generate_article):
     journal_page = checks.JOURNAL.article(id=article.id(), has_figures=article.has_figures())
     assert "Szymon \xc5\x81\xc4\x99ski" in journal_page
 
+@pytest.mark.journal
+@pytest.mark.continuum
+@pytest.mark.bot
+@pytest.mark.lax
+def test_article_google_scholar_metadata(generate_article):
+    article = generate_article(template_id=15893)
+    _ingest_and_publish(article)
+    checks.API.wait_article(id=article.id())
+    google_scholar_page = checks.JOURNAL_CDN.article(id=article.id(), headers={'User-Agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
+    general_page = checks.JOURNAL.article(id=article.id())
+    assert "<meta name=\"citation_" in google_scholar_page
+    assert "<meta name=\"citation_" not in general_page
```

Comment:
Need to look at how to use MechanicalSoup.

Reply:
Not necessary here, as it implements the "browser session" behavior on multiple requests (e.g. sending cookies, submitting forms) but in this case there is only one request per session. Can [use the underlying `BeautifulSoup` directly](https://github.com/elifesciences/elife-spectrum/blob/4f9fb71720b110500fe9df60f009810eee5a415e/spectrum/checks.py#L582-L585).